api/src/api/users/user_schemas.py

```diff
@@ -681,3 +686,55 @@ class UserInvitationListResponseSchema(AbstractResponseSchema):
         fields.Nested(UserInvitationItemSchema),
         metadata={"description": "List of user invitations"},
     )
+
+
+class UserResponseOrgInvitationRequestSchema(Schema):
+    status = fields.Enum(
+        OrganizationInvitationStatus,
+        validate=validators.OneOf(
+            [
+                OrganizationInvitationStatus.ACCEPTED,
+                OrganizationInvitationStatus.REJECTED,
+            ]
+        ),
+        metadata={"description": "User response to invitation"},
+    )
+
+
+class OrganizationSchema(Schema):
+    organization_id = fields.UUID(
+        metadata={
+            "description": "The unique identifier of the organization",
+            "example": "456e7890-e12c-34f5-b678-901234567890",
+        }
+    )
+    organization_name = fields.String(
+        allow_none=True,
+        metadata={
+            "description": "Organization name",
+            "example": "Legal business name of the corresponding Sam Gov Entity",
+        },
+    )
```

Comment:
```suggestion
    organization_name = fields.String(
        allow_none=True,
        attribute="name"
        metadata={
            "description": "Organization name",
            "example": "Legal business name of the corresponding Sam Gov Entity",
        },
    )
```

This should allow `organization_name` to work without explicitly specifying it in the response.